openapi.json

```diff
@@ -6555,6 +6555,9 @@
                         },
                         "following_replied": {
                           "type": "integer"
+                        },
+                        "circles_activity": {
```

Comment:
@oblakeerickson that's a core api? 🤔

Reply:
Comes from 

https://github.com/discourse/discourse/blob/main/spec/requests/api/schemas/json/site_response.json#L131

https://github.com/discourse/discourse/blob/main/app/models/notification.rb#L158


I'm going to merge this in because we have done the same things for several other ENUMS we need to add from plugins.